cocos/core/platform/macro.ts

```diff
@@ -902,12 +903,30 @@ interface Macro {
      */
     ORIENTATION_PORTRAIT: number;
 
+    /**
+     * @en Inverted portrait orientation, with the device bottom facing upward.
+     * @zh 倒置竖屏，设备底部朝上
+     */
+    ORIENTATION_PORTRAIT_UPSIDE_DOWN: number;
+
     /**
      * @en Oriented horizontally
      * @zh 横屏朝向
      */
     ORIENTATION_LANDSCAPE: number;
 
+    /**
+     * @en Oriented horizontally, top side facing to the left.
+     * @zh 水平方向，顶部朝向左侧
+     */
+    ORIENTATION_LANDSCAPE_LEFT: number;
+
+    /**
+     * @en Oriented horizontally, top side facing to the right.
+     * @zh 水平方向，顶部朝向右侧
+     */
+    ORIENTATION_LANDSCAPE_RIGHT: number;
```

Comment:
What's the relationship of `ORIENTATION_LANDSCAPE`, `ORIENTATION_LANDSCAPE_LEFT ` and `ORIENTATION_LANDSCAPE_RIGHT `?

Reply:
When will trigger a `ORIENTATION_LANDSCAPE ` event?

Reply:
> When will trigger a `ORIENTATION_LANDSCAPE ` event?

Called from user, such as view.setOrientation.
Currently, setOrientation only supports landscape, portrait and auto